aten/src/ATen/native/quantized/cpu/qnnpack/src/q8avgpool/up8x9-sse2.c

```diff
@@ -8,7 +8,9 @@
 
 #include <assert.h>
 
+#if defined(__x86_64__) || defined(__i386__) || defined(_MSC_VER)
```

Comment:
Is _MSC_VER necessary?

Reply:
I think we can compile the code under Windows too - I don't think the __x86_64__ is defined there

Reply:
My worry was compiling windows on arm. So something like `defined(_MSC_VER) && defined(_M_IX86)`? I am not sure if thats the right macro though.